src/alignment.js

```diff
@@ -138,12 +138,15 @@ var clearUnvisitedDOM = function(node) {
 
   while (lastChild !== lastVisitedChild) {
     node.removeChild(lastChild);
+    notifications.removedNode(lastChild);
```

Comment:
You will need to check that the node does not have a `key` here as it will cause two removed node notifications currently.